tests/model/test_model.py

```diff
@@ -1940,6 +1942,32 @@ def test__handle_subscription_event_subscribers_one_user_multiple_streams(
             new_subscribers = model.stream_dict[stream_id]['subscribers']
             assert new_subscribers == expected_subscribers
 
+    def test_update_twenty_four_hour_format(self, mocker, model):
+        event = {
+            'type': 'update_display_settings',
+            'setting_name': 'twenty_four_hour_time',
+            'setting': True
+        }
+        twenty_four_hr_format = False
```

Comment:
We can extract the True/False values into parametrized values here, to ensure we test different initial and expected values of the setting after the event.

Reply:
Yes 👍🏼